src/query/builder.js

```diff
@@ -751,11 +755,23 @@ assign(Builder.prototype, {
     let amt = parseInt(amount, 10);
     if (isNaN(amt)) amt = 1;
     this._method = 'counter';
-    this._single.counter = {
-      column,
-      amount: amt,
-      symbol: (symbol || '+')
-    };
+    let counter = this._single.counter || {};
+
+    if(!counter[column]) {
+      counter[column] = {
+        amount: amt,
+        symbol: (symbol || '+')
+      };
+    } else {
+      //Counter for column already exists. Add/subtract the current column amount by the new amount
+      if(symbol === '-') {
+        counter[column].amount -= amount;
```

Comment:
@wubzz Yep. No point of reviewing, just discussing... and this probably is the worst place for that :D 

For that particular use case it would be convenient. But if there are just 0.01% of userbase needing that feature it is hard to justify increase of complexity. 

For example if I'm writing parser that afterwards builds query from AST I probably wouldn't call builder directly from token handlers, but would first create my own internal representation of the query to prevent fight with shortcomings of query builder (also because I would like to normalize the internal presentation before building the final query to make optimization / testing etc. easier).